lib/board.rb

```diff
@@ -32,4 +28,20 @@ def empty_positions
     def position_empty?(position)
         positions[position] == Token::EMPTY
     end
+
+    def positions_with_idx
+        positions.map.with_index { |val, idx| [val, idx] }
+    end
+
+    def center
+        size == 3 ? 4 : nil
+    end
+
+    def corners
+        if size == 3
+            return [0, 2, 6, 8]
+        else
+            return [0, 3, 12, 15]
```

Comment:
Don't need `return`...and can you make this so it's not hard coded by the size of the board?